create_infotexts.py

```diff
@@ -333,42 +344,49 @@ def select_best_mapping_for_depicted_person(self, flipped_name):
         return name_as_wikitext
 
 
-    def process_depicted_place(self, place_string, item):
+    def process_depicted_place(self, place_string, places_mapping):
         """
         Create wikiformat depicted place string from raw input data.
         
         Populates data["depicted_place"] and interacts with data["commonscat"] and data["meta_cats"].
         
-        :param place_string: string value <Ort, foto> in metadata item.  
+        :param place_string: string value <Ort, foto> in metadata item.
+        :param places_mapping: Dictionary containing Commons:Medelhavsmuseet/batchUploads/Cypern_places kw mappings
         :return: None (output stored in object attribute
         """
-        places_mapping = load_places_mapping()
-        # print(places_mapping)
-
-        name_as_wikitext = ""
-
-        if place_string == "":
+        place_as_wikitext = ""
+        if not place_string:
             self.data["depicted_place"] = ""
             return
 
-        elif place_string in places_mapping:
-            if not places_mapping[item["Ort, foto"]]["wikidata"] == "-":
-                self.data["depicted_place"] = "{{city|1={wikidata}|link=wikidata}}".format(
-                    wikidata=places_mapping[item["Ort, foto"]]["wikidata"][2:]
+        if place_string in places_mapping:
+            if not places_mapping[place_string]["wikidata"] == "-":
+                 place_as_wikitext = "{{city|1={wikidata}}}".format(
+                    wikidata=places_mapping[place_string]["wikidata"][2:]
                 )
-            else:
-                self.data["depicted_place"] = "{{city|1={commons}|link=commons}}".format(
-                    commons=places_mapping[item["Ort, foto"]]["commons"]
+            elif place_string == "Stockholm":  # Mainly interiors from buildings gardens
+                self.meta_cats.append("[[Media_contributed_by_SMVK_taken_in_Stockholm]]")
+                self.data["depicted_place"] = ""
```

Comment:
just skip this line (we already have `place_as_wikitext = ""` before) or use `place_as_wikitext = place_string` here since Stockholm is still true (just no as specific as we would want it in the end).